tests/java-db-test/mysql/create.sql

```diff
@@ -0,0 +1,104 @@
+CREATE DATABASE mintern;
```

Comment:
Seems to be the same as the file in the other PR. Apply those changes here as well :)

Reply:
done